pallets/fees/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+authors = ["Centrifuge <admin@centrifuge.io>"]
+description = 'Fees pallet for runtime'
+edition = '2018'
+license = "LGPL-3.0"
+name = 'fees'
+repository = "https://github.com/centrifuge/centrifuge-chain/pallets/fees"
+version = '2.0.0'
+
+[package.metadata.docs.rs]
+targets = ['x86_64-unknown-linux-gnu']
+
+# alias "parity-scale-code" to "codec"
+[dependencies.codec]
+default-features = false
+features = ['derive']
+package = 'parity-scale-codec'
+version = '2.0.0'
+
+[dependencies]
+frame-support = { git = "https://github.com/centrifuge/substrate", default-features = false , branch = "master" }
+frame-system = { git = "https://github.com/centrifuge/substrate", default-features = false , branch = "master" }
+sp-runtime = { git = "https://github.com/centrifuge/substrate", default-features = false , branch = "master" }
+pallet-authorship = { git = "https://github.com/centrifuge/substrate",  default-features = false , branch = "master" }
+pallet-balances = { git = "https://github.com/centrifuge/substrate",  default-features = false , branch = "master" }
+
+[dev-dependencies]
+sp-core = { git = "https://github.com/centrifuge/substrate", default-features = false , branch = "master" }
+sp-io = { git = "https://github.com/centrifuge/substrate", default-features = false , branch = "master" }
+
+[features]
+default = ['std']
+std = [
+    'codec/std',
+    'frame-support/std',
+    'frame-system/std',
+    'pallet-authorship/std',
+    'pallet-balances/std'
+]
```

Comment:
Preparing for runtime benchmarking now that we refactor, adding:

runtime-benchmarks = ["frame-benchmarking"]

Reply:
we can do this once we start working on benchmarks. I would prefer it as it :)